vendor/github.com/quobyte/api/quobyte.go

```diff
@@ -5,20 +5,38 @@ import (
 	"net/http"
 )
 
+// retry policy codes
+const (
+	RetryNever         string = "NEVER"
+	RetryInteractive   string = "INTERACTIVE"
+	RetryInfinitely    string = "INFINITELY"
+	RetryOncePerTarget string = "ONCE_PER_TARGET"
+)
+
 type QuobyteClient struct {
-	client   *http.Client
-	url      string
-	username string
-	password string
+	client         *http.Client
+	url            string
+	username       string
+	password       string
+	apiRetryPolicy string
+}
+
+func (client *QuobyteClient) SetAPIRetryPolicy(retry string) {
+	client.apiRetryPolicy = retry
+}
+
+func (client *QuobyteClient) GetAPIRetryPolicy() string {
+	return client.apiRetryPolicy
 }
 
 // NewQuobyteClient creates a new Quobyte API client
 func NewQuobyteClient(url string, username string, password string) *QuobyteClient {
 	return &QuobyteClient{
-		client:   &http.Client{},
-		url:      url,
-		username: username,
-		password: password,
+		client:         &http.Client{},
+		url:            url,
+		username:       username,
+		password:       password,
+		apiRetryPolicy: RetryInteractive,
```

Comment:
Default INTERACTIVE user can change it with setter method. [Set retry policy for the client](https://github.com/kubernetes/kubernetes/pull/54631/files#diff-b934840b7581a8b98216417877abf713)

Reply:
ONCE_PER_TARGET is configurable. Would you suggest it making default?

client := quobyte_api.NewQuobyteClient("http://apiserver:7860", "user", "password")
client.SetAPIRetryPolicy(quobyte_api.RetryOncePerTarget)

Reply:
Actually it's not configurable it's hardcoded in the code base (since this part will use always the default) so to "configure" you would need to compile Kubernetes by yourself. 

I think `ONCE_PER_TARGET` is a better default than `INTERACTIVE` (or what is the concrete difference here?).

Optional we could make this in an additional MR configurable for each StorageClass.

Reply:
Felix said retry setting is going to be a configurable option soon.

The difference is ONCE_PER_TARGET ensures all the targets hit at least once, INTERACTIVE returns on the first error. Both of these, retry the at least one more time if the error is retryable. Some retryable errors are timeout, device not available, unresolvable endpoints etc.